templates/keycloak_provision

```diff
@@ -20,7 +20,7 @@ if [ $? -eq 0 ]; then echo "Realm exists"; exit 0; fi \
                                         -s clientId=$keycloakClientId \
                                         -s id=$keycloakClientId \
                                         -s 'webOrigins=["http://$cheHost", "https://$cheHost"]' \
-                                        -s 'redirectUris=["http://$cheHost/*", "https://$cheHost/*"]' \
+                                        -s 'redirectUris=["http://$cheHost/dashboard/*", "https://$cheHost/dashboard/*", "http://$cheHost/workspace-loader/*", "https://$cheHost/workspace-loader/*", "http://$cheHost/_app/*", "https://$cheHost/_app/*"]' \
```

Comment:
Not really related to the current PR but can we instead of duplication http and https redirects just provide the right protocol via parameter as we provide $cheHost? It would help us be more secure and make sure token if not send through http when https is configured

Reply:
Agree